custom_components/better_thermostat/models/utils.py

```diff
@@ -75,14 +75,14 @@ def calculate_local_setpoint_delta(self) -> Union[float, None]:
 	the external sensor.
 	"""
 	
-	state = self.hass.states.get(self.heater_entity_id).attributes
-	
+	trvState = self.hass.states.get(self.heater_entity_id).attributes
+	calibrationState =  self.hass.states.get(self.local_temperature_calibration_entity).state
 	_context = "calculate_local_setpoint_delta()"
 	
-	_current_trv_temp = convert_to_float(state.get('current_temperature'), self.name, _context)
-	_current_trv_calibration = convert_to_float(state.get('local_temperature_calibration'), self.name, _context)
+	_current_trv_temp = convert_to_float(trvState.get('current_temperature'), self.name, _context)
+	_current_trv_calibration = convert_to_float(calibrationState, self.name, _context)
 	
-	if not all([_current_trv_calibration, self._cur_temp, _current_trv_temp]):
+	if any(check is None for check in [_current_trv_calibration, self._cur_temp, _current_trv_temp]):
```

Comment:
@RubenKelevra Check this out,

we need to test if any of this values is None, but go through if its 0

Reply:
We shoud check if we need this kind of check mod on other places where 0 is also valid

Reply:
So an int at 0 gives `False`. Wasn't expecting that. But makes some sense, as `bool(0)` is also False as well as `if 0:`.

We can do `None in [List, of, variables]`:

```
>>> a = None
>>> b = 0
>>> c = 1
>>> None in [a, b, c]
True
>>> a = 0
>>> None in [a, b, c]
False
```

Reply:
I feel like that's better readable. Can you change the code? :)

Reply:
#355 fixes the rest of the codebase. I used tuple so () instead of [].